dragonfly/block/endstone.go

```diff
@@ -0,0 +1,34 @@
+package block
+
+import "github.com/df-mc/dragonfly/dragonfly/item"
+
+// Endstone is a block found in The End.
+type Endstone struct {
```

Comment:
The Minecraft wiki refers to this as `End Stone` so it might be better to call this type `EndStone` (and the file `end_stone.go`)